README.md

```diff
@@ -139,106 +78,19 @@ To amend the round dates
     docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.amend_round_dates --round_id c603d114-5364-4474-a0c4-c41cbf4d3bbd --opens_date "2022-10-04 12:00:00" --deadline_date "2022-12-14 11:59:00" --assessment_deadline_date "2023-03-30 12:00:00"
 ```
 
-To truncate data before re-loading it run
-
-```bash
-    docker exec -it $(docker ps -qf "name=fund-store") inv truncate-data
-```
-
-### Create and seed local DB
-- Make sure your local `DATABASE_URL` env var is set to your local postgres db (this doesn't need to actually exist yet), eg:
-
-```bash
-    # pragma: allowlist nextline secret
-    DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/fsd_fund_store
-```
-
-- Create and seed using the following scripts:
-
-```bash
-        python -m scripts.fund_round_loaders.{load_config_script}
-```
-
-### Build with Paketo
-
-[Pack](https://buildpacks.io/docs/tools/pack/cli/pack_build/)
-
-[Paketo buildpacks](https://paketo.io/)
-
-```bash
-    pack build <name your image> --builder paketobuildpacks/builder:base
-```
-
-Example:
-
-    [~/work/repos/funding-service-design-fund-store] pack build paketo-demofsd-app --builder paketobuildpacks/builder:base
-    ***
-    Successfully built image paketo-demofsd-app
-
-
-You can then use that image with docker to run a container
-
-```bash
-    docker run -d -p 8080:8080 --env PORT=8080 --env FLASK_ENV=dev [envs] paketo-demofsd-app
-```
-
-`envs` needs to include values for each of:
-SENTRY_DSN
-GITHUB_SHA
-
-```bash
-    docker ps -a
-    CONTAINER ID   IMAGE                       COMMAND                  CREATED          STATUS                    PORTS                    NAMES
-    42633142c619   paketo-demofsd-app          "/cnb/process/web"       8 seconds ago    Up 7 seconds              0.0.0.0:8080->8080/tcp   peaceful_knuth
-```
-
-# Pipelines
-
-Place brief descriptions of Pipelines here
-
-* Tag to Release - This is a simple pipeline that will create the tag required and push an image, built with Paketo, to the GHCR with that tag
-
 # Testing
+[Testing in Python repos](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-db-development.md)
```

Comment:
We could get this to embed using [pandoc](https://pandoc.org/filters.html#include-files) but is it worth the extra effort when a click will navigate you through? I would say no

Reply:
Yeah, I agree.